spyder/plugins/appearance/confpage.py

```diff
@@ -321,27 +319,32 @@ def update_preview(self, index=None, scheme_name=None):
         """
         text = ('"""A string"""\n\n'
                 '# A comment\n\n'
-                '# %% A cell\n\n'
```

Comment:
Why are we not displaying cells in the preview anymore? That's a very important functionality, so I don't agree with removing them.

Reply:
It is not an important functionality for a simple code editor. Having cells appear in the appearance settings is completely secondary. I do not agree with implementing that.